ui/app/styles/components/list-item-row.scss

```diff
@@ -46,8 +46,7 @@ a.list-item-row,
     padding-left: $size-9;
     padding-right: $size-9;
     position: relative;
```

Comment:
removed the `box-link-xx` vars. If we had used it consistently, I could see keeping it.